src/library.js

```diff
@@ -784,8 +784,20 @@ LibraryManager.library = {
     abort('Assertion failed: ' + (condition ? UTF8ToString(condition) : 'unknown condition') + ', at: ' + [filename ? UTF8ToString(filename) : 'unknown filename', line, func ? UTF8ToString(func) : 'unknown function']);
   },
 
-  terminate__sig: 'vi',
-  terminate: '__cxa_call_unexpected',
+  __cxa_call_unexpected: function(exception) {
+    err('Unexpected exception thrown, this is not properly supported - aborting');
+#if !MINIMAL_RUNTIME
+    ABORT = true;
+#endif
+    throw exception;
+  },
+
+  terminate: function() {
+#if !MINIMAL_RUNTIME
+    ABORT = true;
+#endif
+    throw 'terminated';
```

Comment:
Nice, but we we have any code that even calls this function?   I don't know there is something called `std::terminate` in c++ but this function here is not mangled.?